op-chain-ops/cmd/celo-migrate/beacon.go

```diff
@@ -60,10 +59,16 @@ func AwaitEpoch(chainID *big.Int, epoch uint64) {
 // findL1StartingBlock looks back or forward (depending on the epochIncrement)
 // for a finalized L1 block that is up to maxSequencerDrift before the L2 fork
 // block.
-func (c *BeaconClient) findL1StartingBlock(chainID *big.Int, unixTime uint64, epochIncrement int64) (common.Hash, error) {
+func (c *BeaconClient) findL1StartingBlock(chainID uint64, unixTime uint64, epochIncrement int64) (common.Hash, error) {
 	var l1StartBlockHash common.Hash
 	var l1StartBlockTime uint64
-	for epoch := int64(ContainingEpoch(chainID, unixTime)); ; epoch += epochIncrement {
+
+	epoch := int64(ContainingEpoch(chainID, unixTime))
```

Comment:
```suggestion
	epoch := ContainingEpoch(chainID, unixTime)
```

Reply:
We need an int64 epoch here because it gets added to the increment which is provided as an int64 so that it can be negative.